pkg/config/config.go

```diff
@@ -1164,6 +1164,7 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("capture_lambda_payload", false)
 	config.BindEnvAndSetDefault("serverless.trace_enabled", false, "DD_TRACE_ENABLED")
 	config.BindEnvAndSetDefault("serverless.trace_managed_services", true, "DD_TRACE_MANAGED_SERVICES")
+	config.BindEnvAndSetDefault("serverless.shell_port", 5555)
```

Comment:
It might not be a concern with serverless but you might as well use a port which is not used by any other known service (cf https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers).